src/www/ui/api/documentation/openapi.yaml

```
@@ -4418,6 +4444,97 @@ components:
               enum:
                 - OK
                 - ERROR
+    confResponse:
```

Please following naming convention
```suggestion
    ConfResponse
```